packages/gatsby-source-nacelle/package.json

```diff
@@ -27,9 +27,14 @@
     "lodash.set": "^4.3.2"
   },
   "peerDependencies": {
-    "gatsby": ">2.0.31",
-    "gatsby-source-filesystem": "^2.6.1",
-    "graphql": "^14.7.0"
+    "gatsby": "^4.0.0",
+    "gatsby-source-filesystem": "^4.0.0",
```

Comment:
I know we were using `>` before, but after the v4 breaking changes stuff I kind of figured we need to evaluate each major version on a case by case basis and update the dependencies appropriately.

Reply:
I think that's the right call, given what we've learned in recent weeks.